### PR TITLE
Add documentation for mail app

### PR DESCRIPTION
**Pull request by GretaD**
ref #9908 
Screenshot from 2023-05-02 17-56-59

### CODE REVIEW

**Comment by GretaD** [comment]
@ChristophWurst please have a general look if this is good enough for a first draft

**Comment by ChristophWurst** [comment]
You should be able to build the documentation locally after https://github.com/nextcloud/documentation/pull/9967

File: user_manual/groupware/mail.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Shared mailbox
+~~~~~~~~~~~~~~~
+If a mailbox was shared with you with some specific rights, that mailbox will show as a new mailbox with a shared icon as below:
+
+.. figure:: images/shared-mailbox-icon.png
+
+Envelope actions
+~~~~~~~~~~~~~~~~~
+Create an event
+~~~~~~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (GretaD) [reply user_manual/groupware/mail.rst:81]: Right, create event is a sub-element of envelope action, will fix it asap. Have a look at general content so i can fix everything in one fixup

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply user_manual/groupware/mail.rst:81]: Fine for a first version

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**